### ISSUE TITLE
For OpenWrt release 22.03.2

### ISSUE THREAD

**Issue by cmangla** [openwrt/gh-action-sdk#32]
This action seems to work for `ARCH=23.05.0-*` and `ARCH=23.05.2-*` but not `ARCH=22.03.2-*`.

What has changed since 22.03.2?

How do I fix this, either in the way I use this action or in this action itself?

**Comment by cmangla** [comment]
Consider this change, in which I try to add 22.03.2 to a github action which works fine for 23.05.0:

https://github.com/cmangla/AirSane-openwrt/commit/72eede592271481db36ad979ff2418ba4df61c65

See the lines around the link above to see how I'm using this action. The build fails for the above commit are available at:

https://github.com/cmangla/AirSane-openwrt/actions/runs/6998038264/job/19035663379#step:13:59

and:

https://github.com/cmangla/AirSane-openwrt/actions/runs/6998038264/job/19035663442

So now I have two jobs, one for 22.03.2 and another one for 23.*. You can see the job definition that works for 22.03.2 at:

https://github.com/cmangla/AirSane-openwrt/blob/f078742cf048f3858682dacaee330d4a650ec6df/.github/workflows/build.yml#L12

In the above file, I have a job called 'build' which is for 22.03.2, which works using custom steps, and below that a job called 'build_post_r23' which uses this action for 23.05.2 and 23.05.0 but not 22.03.2.

My suspicions are on the `working-directory` setting in the job named 'build'. In the `openwrt/Docker` repository, they set `WORKDIR` to `/build`. See:

https://github.com/openwrt/docker/blob/f3c55936b9c8915e3c3fd6ec8ffb24f2d372a4d4/Dockerfile#L5

**Comment by cmangla** [comment]
Ok, now that I reviewed the build failure when writing the comment above, I noticed the error in the output:

```
ERROR: failed to solve: ghcr.io/openwrt/sdk:bcm27xx-bcm2708-22.03.2: ghcr.io/openwrt/sdk:bcm27xx-bcm2708-22.03.2: not found
```

So this action cannot find the images. But the 'build' job is able to.

**Comment by cmangla** [comment]
I suppose I can fix this by changing the `CONTAINER` variable for older builds to just `openwrt/sdk`. Or perhaps I can set CONTAINER to that for all builds?

**Comment by aparcar** [comment]
I recreated the containers for 22.03.2 can you please try again?

**Comment by cmangla** [comment]
Yes, the containers for 22.03.2 now seem to work, when them being fetched from `ghcr.io`. Thanks for your help!
